src/App.vue

```diff
@@ -228,26 +228,12 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
   /** The Update Current JS Date timer id. */
   private updateCurrentJsDateId = 0
 
-  /** The entity title. */
-  get entityTitle (): string {
-    switch (this.$route.name) {
-      case RouteNames.ALTERATION: return 'Company Information'
-      case RouteNames.CHANGE: return 'Business Information'
-      case RouteNames.CONVERSION: return 'Record Conversion'
-      case RouteNames.CORRECTION: return 'Register Correction'
-      case RouteNames.RESTORATION_EXTENSION: return 'Limited Restoration Extension'
-      case RouteNames.RESTORATION_CONVERSION: return 'Conversion to Full Restoration'
-      case RouteNames.SPECIAL_RESOLUTION: return 'Special Resolution'
-    }
-    return 'Unknown Filing' // should never happen
-  }
```

Comment:
Moved into `routes.ts` (and updated breadcrumbs below), making 1 less place to change when adding a filing.